### PR TITLE
Multi-Tenant performance pack 

### PR DESCRIPTION
**Pull request by davidbinyamin**
## Status
- [x] Ready

## Description
Pack that contains widgets and dashboard to monitor MT hosts and host groups health statistics

## Screenshots
Host groups dashboard:
groupDash

hosts dashboard:
hostsDash

## Minimum version of Demisto
- [x] 6.1.0 (beta??)

## Does it break backward compatibility?
   - [x] No

@guydemi FYI

### CODE REVIEW

**Comment by davidbinyamin** [comment]
BTW, i am not sure master/6.0.0 tests should pass at all.
this pack minimum server version is 6.2.0.
@IkaDemisto @guykeller

**Review by IkaDemisto — APPROVED** [review]
Cool pack 📦 
How can I install it on our feature branch from without modifying `FromVersion` ? 
Just want to verify that pack installation is o.k from marketplace.

File: Packs/MultiTenantPerformance/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Multi-Tenant Performance",
+    "description": "Multi-Tenant related content to monitor hosts and hosts groups performance",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "serverMinVersion": "6.2.0",
```
Comment (IkaDemisto) [inline Packs/MultiTenantPerformance/pack_metadata.json:8]: Need to make sure that the pack isn't merged until it's final

**Comment by davidbinyamin** [comment]
> Cool pack 📦
> How can I install it on our feature branch from without modifying `FromVersion` ?
> Just want to verify that pack installation is o.k from marketplace.

No, you’ll have to change the min version

**Comment by davidbinyamin** [comment]
@IkaDemisto what about the master/6_0 tests failures?
it is the required behaviour, to not show the pack with advanced version

**Comment by IkaDemisto** [comment]
> @IkaDemisto what about the master/6_0 tests failures?
> it is the required behaviour, to not show the pack with advanced version

@davidbinyamin I will open dev task on it, still not supported in content build.

**Comment by davidbinyamin** [comment]
@IkaDemisto changed the min version to 6.1.0-beta.
so the only failed test is on 6.0.
BTW, you can now install it on your "master" env

File: Packs/MultiTenantPerformance/Dashboards/dashboard-Host_Groups_Health.json [hunk 2]
```diff
@@ -0,0 +1,495 @@
+{
+    "id": "Host Groups - Health",
+    "version": -1,
+    "fromVersion": "6.1.0-beta",
```
Comment (IkaDemisto) [inline Packs/MultiTenantPerformance/Dashboards/dashboard-Host_Groups_Health.json:4]: Lets remove the `beta` prefix and leave it as `6.1.0` or `6.2.0`

File: Packs/MultiTenantPerformance/Dashboards/dashboard-Host_Groups_Health.json [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+            "widget": {
+                "id": "ha-group-cpu-bar",
+                "version": 1,
+                "sequenceNumber": 17755,
+                "primaryTerm": 1,
+                "modified": "2020-11-17T09:41:35.38135Z",
+                "sortValues": null,
+                "packID": "MultiTenantPerformance",
+                "itemVersion": "1.0.0",
+                "fromServerVersion": "5.0.0",
```
Comment (IkaDemisto) [inline Packs/MultiTenantPerformance/Dashboards/dashboard-Host_Groups_Health.json:34]: Note that there are several places with `fromServerVersion` with 5.0.0 version

**Comment by IkaDemisto** [comment]
@davidbinyamin please pull from master, the issue should be fixed now.

**Comment by yaron-libman** [comment]
Reviewed and approved.

**Pull request merged**